dspace-rest/src/main/java/org/dspace/rest/ItemsResource.java

```diff
@@ -992,7 +992,7 @@ else if (!AuthorizeManager.authorizeActionBoolean(context, bitstream, org.dspace
             {
                 sql += "QUALIFIER='" + metadata[2] + "' AND ";
             }
-            sql += "dbms_lob.substr(TEXT_VALUE, 40)='" + metadataEntry.getValue() + "' AND ";
+            sql += "TEXT_VALUE='" + metadataEntry.getValue() + "' AND ";
```

Comment:
It would probably be best to construct that part of SQL separately for Oracle and Postgres.

The solution for Oracle doesn't seem ideal, isn't there a way to do it without a constant? If this is the best we can do, I'll update the code here.